File: tessoku_ch1/a2.py
n,x = map(int, input().split())
a = list(map(int, input().split()))

if x in a:
    print("Yes")
    
answer = False

for i in range(n):
    if a[i] == x:
        answer = True
        
if answer:
    print("Yes")
        

File: tessoku_book/a2.cpp
#include <iostream>
#include <vector>
#include<algorithm>
using namespace std;

int main(){
    int n,x,a[109];
    cin>>n>>x;
    bool answer=false;
    for(int i=1;i<=n;i++)
    {
        cin>>a[i];
        if(x==a[i])answer=true;
    }
    if(answer==true)cout<<"Yes"<<endl;
    else cout<<"No"<<endl;
    return 0;
}